### ISSUE TITLE
Have crosstab print both percent and n?

### ISSUE THREAD

**Issue by sfirke** [sfirke/janitor#29]
- [x] Add appropriate number of 0s in percentages, e.g., 0.0% instead of 0%

Not useful for subsequent analysis but handy for examining, and sometimes for sharing.  So something like specifying `percent = "row", mode = "combined"` and it would have in each cell a value like `10.4% (31)`.

Don't want to excessively complicate things, and it raises the issue of truncation preferences.  Having parameters for `percent`, `mode`, and `digits` seems excessive.

I could have a separate function that handles all % related calculations, like:
 `crosstab(mtcars$cyl, mtcars$gear) %>% table_percentages(., percent = "row", show = "combined", digits = 1)`
Though I don't like taking the simple percentage option out of `crosstab()`.  But if I don't, there's redundancy.

Might just write this as a personal function.  But I have to imagine it's widely used.

**Comment by sfirke** [comment]
Or if a separate function would always be called after `crosstab()`, it might as well wrap it.  So `hybrid_crosstab(mtcars$cyl, mtcars$gear, percent = "row", show_na = "TRUE", digits = 1)` would just be a simple wrapper of crosstab().

I also don't like having multiple functions doing pretty similar things...

**Comment by sfirke** [comment]
I would want a `rounding_method` argument if truncating, so that there's "round to even" and "round ties up."

Leaning toward a separate function `format_crosstab()` that would take the result of a `crosstab` call and add custom truncation of %s, with or without parenthetical Ns, and support rounding ties up.
```
crosstab(mtcars$cyl, mtcars$gear) %>%
  format_crosstab(., denom = "row", show_n = TRUE, digits = 2, rounding = "round ties up")
```

**Comment by sfirke** [comment]
Yeah, I need this.  No good to put:
```
  mutate_at(vars(2:7), funs(. * 100)) %>%
  mutate_at(vars(2:7), funs(round(., 1))) %>%
  mutate_at(vars(2:7), funs(paste0(., "%"))) %>%
  ```

Everytime I call `kable()` on the output of `crosstab()`.

**Comment by sfirke** [comment]
A working version is live on the prettify_crosstab branch.  Needs tests.  Should it be called "adorn_crosstab" or the more generic "format_crosstab"?

**Comment by sfirke** [comment]
Fully implemented in f3a6c318a66f165611a4f9beede57f4fe0a332d2
